gazelle/common/treesitter/grammars/BUILD.bazel

```diff
@@ -0,0 +1,2 @@
+# gazelle:ignore *.bzl
+# gazelle:lang off
```

Comment:
For some reason gazelle removes the required `go_library(copts)` option in the aspect-cli repo. I don't understand why it happens in this repo but not the internal one.